lametro/views.py

```diff
@@ -299,7 +301,9 @@ def get_context_data(self, **kwargs):
 
         # If all meetings
         elif self.request.GET.get('show'):
-            all_events = LAMetroEvent.objects.order_by('-start_time')
+            all_events = LAMetroEvent.objects\
+                                     .order_by('-start_time')\
+                                     .prefetch_related('media_urls')
```

Comment:
do we see the same gains if we use the `Prefetch` object to return `LAMetroEventMedia` objects? this works in the shell:

```python
from django.db.models import Prefetch

LAMetroEvent.objects.filter(SOME FILTER).prefetch_related(Prefetch('media_urls', queryset=LAMetroEventMedia.objects.all()))
```

Reply:
We do get the same gains! Building off of that....do you think it's too abstruse to order the Event Media when prefetching? I implemented how that would work here:

https://github.com/datamade/la-metro-councilmatic/pull/406/files#diff-803b5091212eac625d7bc1c2dd933c21R322